app/views/buildings/show.html.erb

```diff
@@ -5,5 +5,17 @@
     <li><%= link_to suite.number, suite_path(suite) %> - <%= pluralize(suite.size,'bed') %></li>
   <% end %>
 </ul>
+<h2>Import Suites</h2>
+<p>Add many suites at once by uploading a CSV file with the building information.</p>
+<p>The CSV must have the format "Number, Singles, Doubles" where Number is the 
+identifier of the suite, Singles is the number of single bedrooms in the suite,
+and Doubles is the number of double bedrooms in the suite.</p>
+<p>
+<%= simple_form_for @suite_importer, url: suite_import_path(@suite_importer) do |f| %>
```

Comment:
Why are you passing `@suite_importer` to `suite_import_path`? I don't think it needs any parameters to generate the route.